alf/environments/suite_go_test.py

```diff
@@ -275,21 +387,81 @@ def test_environment(self):
                          torch.tensor([0, 0, 0], dtype=torch.int8))
         self.assertEqual(time_step.observation['board'][0:2],
                          torch.zeros(2, 1, 4, 4))
-        # board does not change after pass
-        self.assertEqual(time_step.observation['board'][2],
-                         torch.tensor(sequences[2][-1][2]).unsqueeze(0))
+        # board[2] does not change after pass
+        self.assertEqual(time_step.observation['board'][2][0],
+                         sequences[2][-1]['board'])
 
-        # Both pass
+        # board[2] both pass
         time_step = env.step(
             action=torch.tensor([4, 5, 16], dtype=torch.int64))
         self.assertEqual(
             time_step.step_type,
             torch.tensor([StepType.MID, StepType.MID, StepType.LAST]))
-        # player 0 wins
-        self.assertEqual(time_step.reward, torch.tensor([0., 0., 1.]))
+        # player 1 wins
+        self.assertEqual(time_step.reward, torch.tensor([0., 0., -1.]))
         # board does not change after pass
-        self.assertEqual(time_step.observation['board'][2],
-                         torch.tensor(sequences[2][-1][2]).unsqueeze(0))
+        self.assertEqual(time_step.observation['board'][2][0],
+                         sequences[2][-1]['board'])
+
+    def test_environment2(self):
```

Comment:
Maybe some comments here (and also test_environment) to tell the difference between to environment tests.